src/transport/msg_providers/data_bus.py

```diff
@@ -0,0 +1,281 @@
+import abc
+import logging
+from collections import deque
+from enum import Enum
+from typing import List, Optional
+
+import variables
+from eth_account.account import VRS
+from eth_typing import HexStr
+from schema import Schema
+from transport.msg_providers.common import BaseMessageProvider
+from transport.msg_providers.rabbit import MessageType
+from transport.msg_types.deposit import DepositMessage
+from transport.msg_types.pause import PauseMessage
+from transport.msg_types.ping import PingMessageDataBus
+from transport.msg_types.unvet import UnvetMessage
+from utils.bytes import bytes_to_hex_string
+from web3 import Web3
+from web3._utils.events import get_event_data
+from web3.exceptions import BlockNotFound
+from web3.types import FilterParams, LogReceipt
+
+logger = logging.getLogger(__name__)
+
+_MESSAGE = {
+    'anonymous': True,
+    'inputs': [
+        {'indexed': True, 'name': 'eventId', 'type': 'bytes32'},
+        {'indexed': True, 'name': 'sender', 'type': 'address'},
+        {'indexed': False, 'name': 'data', 'type': 'bytes'},
+    ],
+    'name': 'Message',
+    'type': 'event',
+}
+
```

Comment:
Add as a comment decryption for this ABIs.

Reply:
Done